noxfile.py

```diff
@@ -0,0 +1,59 @@
+# https://cjolowicz.github.io/posts/hypermodern-python-02-testing/
+# https://cjolowicz.github.io/posts/hypermodern-python-03-linting/
+import os
+from typing import Any
+import tempfile
+
+import nox
+from nox.sessions import Session
+
+
+@nox.session(python=["3.6", "3.7", "3.8"])
```

Comment:
Do we actually need any kind of virtual environment here? Because the whole system is already running in the GH Actions environment right? Shouldn't we tell:

`
If you want to disable virtualenv creation altogether, you can set python to False, or set venv_backend to "none", both are equivalent. Note that this can be done temporarily through the –no-venv commandline flag, too.

@nox.session(python=False)
def tests(session):
    pass
`

Otherwise simply use "@nox.session"

Reply:
Thanks for getting into the details of Nox more than I have been able to. I appreciate your suggestion and will try to implement it within the next iteration.